lib/ui/drawing_space/tool_options.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:paintroid/ui/drawing_space/bottom_brush_tool_options.dart';
+import 'package:paintroid/ui/drawing_space/top_brush_tool_options.dart';
+
+class ToolOptions extends ConsumerStatefulWidget {
```

Comment:
Why can't this just be a stateless widget?

Reply:
Good eye. This was done for testing purposes and forgot to change it back again to StatelessWidget. But in the future this widget will be ConsumerWidget. For now it can stay StatelessWidget.